lib/combineNodes.js

```diff
@@ -0,0 +1,367 @@
+'use strict';
+var Cesium = require('cesium');
+var jp = require('jsonpath');
+var _ = require('underscore');
```

Comment:
Unfortunately, yes.  It's the same idea as using `$` for jquery, which I also hate.  That being said, I would strongly suggest not depending on underscore, there is absolutely no reason that I know of to use it over native JavaScript functions.